src/client/modals/source-modal/highlighter.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import jsonLanguage from "highlight.js/lib/languages/json";
+import React from "react";
+import SyntaxHighlighter from "react-syntax-highlighter/src/light";
+import githubGist from "react-syntax-highlighter/src/styles/hljs/github-gist";
+
+SyntaxHighlighter.registerLanguage("json", jsonLanguage);
+
+interface Props {
+}
+
+export default class Highlighter extends React.Component<Props> {
+    render() {
+        const { children: source } = this.props;
+
+        return <SyntaxHighlighter className="source-modal__source" language="json" style={githubGist}>
+            {source}
+        </SyntaxHighlighter>;
+    }
+}
```

Comment:
```suggestion
export const Highlighter: SFC = ({ children: source }) =>
  <SyntaxHighlighter className="source-modal__source" language="json" style={githubGist}>
      {source}
  </SyntaxHighlighter>;
```

Reply:
Of course, need to import SFC from React.

Reply:
TS told me `SFC` is deprecated and I should use `FunctionComponent`.
Also it has to export default to work as lazy component.